examples/gno.land/r/demo/bugs/var_prev_realm/var_prev_realm_test.gno

```diff
@@ -0,0 +1,21 @@
+package var_prev_realm
+
+import (
+	"std"
+	"testing"
+)
+
+func TestVarPrevRealm(t *testing.T) {
+	if initRealm.Addr() != varRealm.Addr() {
+		t.Errorf("initRealm != varRealm")
```

Comment:
thanks for the issue and the PR, it would be nice if your test reports something like 

```suggestion
		t.Errorf("expect initRealm  '%s' to be equal to varRealm '%s'", initRealm.Addr(), varRealm.Addr() )
```

Same for following `t.Errorf`, thanks in advance!

Reply:
done